src/document/utils.js

```diff
@@ -370,12 +370,38 @@ function inheritLabel(doc, fn) {
     : fn(doc);
 }
 
+/**
+ * returns true iff cleanDoc(doc) === ""
```

Comment:
```suggestion
 * returns true if cleanDoc(doc) === ""
```

Reply:
It's intentional. I mean "if and only if"
https://en.wikipedia.org/wiki/If_and_only_if
> "if and only if" (often shortened as "iff")

Reply:
TIL